tests/test_aqmon.py

```diff
@@ -0,0 +1,267 @@
+import os.path
+from unittest import TestCase, skipIf
+import numpy as np
+import strax
+import straxen
+import shutil
+from straxen.test_utils import nt_test_run_id as test_run_id_nT
+
+
+class DummyAqmonHits(strax.Plugin):
+    """
+    Dummy plugin to make some aqmon hits that may span ON and OFF
+    signals over chunks
+
+    There are two channels, that signify ON or OFF of a logic signal
+
+    We are interested in the deadtime, which is the time difference
+    between ON and OFF singals. This plugin computes that deadtime, even
+    if we have missed the first ON (so we start with OFF) or missed the
+    last OFF (such that the last signal is an ON). If we miss one ON at
+    the start or one OFF at the end, we should just consider all time
+    before the first OFF or all the time after the last ON as deadtime.
+    """
+    vetos_per_chunk = strax.Config(
+        # Strax does not like numpy, let's give it a list of integers
+        default=[int(n) for n in np.arange(1, 10)],
```

Comment:
FYI: `list(range(10))` does the same.

Reply:
`list(range(1,10))` if we're being pedantic

Reply:
yeah, you can probably imagine why I wrote it like this instead of `list(range(1,10))`